source/stringified.d.ts

```diff
@@ -0,0 +1,20 @@
+/**
+Create a stringified type of an object.
```

Comment:
Please spend some more time on the description.

Also per PR guidelines, you need to include some use-cases in text form.